src/fides/api/util/aws_util.py

```diff
@@ -0,0 +1,67 @@
+from typing import Any, Dict, Optional
+
+from boto3 import Session
+from botocore.exceptions import ClientError
+from loguru import logger
+
+from fides.api.common_exceptions import StorageUploadError
+from fides.api.schemas.storage.storage import AWSAuthMethod, StorageSecrets
+
+
+def get_aws_session(
```

Comment:
more appropriately/generically named function now. the function here can and should be used by some of our other AWS connectors/codepaths, since this seems like a pretty solid authentication paradigm to promote across AWS resource types.